examples/hello-openshift/v1beta3/hello-project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "Project",
+  "metadata": {
+    "annotations": {
+      "description": "This is an example project to demonstrate OpenShift v3",
+      "displayName": "Hello OpenShift",
```

Comment:
The comma at the end of this line makes it invalid JSON.

A JSON block cannot have a comma after the last key.